auth/src/main/java/com/firebase/ui/auth/ui/email/CheckEmailFragment.java

```diff
@@ -96,6 +99,21 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
         }
 
         view.findViewById(R.id.button_next).setOnClickListener(this);
+
+        TextView termsText = view.findViewById(R.id.email_tos_and_pp_text);
+        TextView footerText = view.findViewById(R.id.email_footer_tos_and_pp_text);
+        FlowParameters flowParameters = getFlowParams();
```

Comment:
Can we get rid of `flowParameters`? (It's just as long as `getFlowParams()` and is only used once.)

Reply:
Done.

Reply:
Nope, SGTM